.github/workflows/ci.yml

```diff
@@ -53,3 +53,38 @@ jobs:
 
       - name: Run tests
         run: pnpm coverage
+
+      # - name: Setup TeX Live
+      #   uses: teatimeguest/setup-texlive-action@v3
+      #   with:
+      #     packages: scheme-basic
+
+      - name: Install texlive-latex
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y texlive-latex-base texlive-xetex texlive-latex-extra texlive-fonts-recommended texlive-fonts-extra
+          xetex -v
+
+      - name: Install fonts
+        run: |
+          sudo apt install fonts-linuxlibertine
```

Comment:
I think we can remove `apt install fonts-linuxlibertine` here because linux libertine font is already included in `texlive-fonts-extra`